src/MessagingThreadListItem/MessagingThreadListItem.tsx

```diff
@@ -69,10 +69,10 @@ export const MessagingThreadListItem: React.FC<
 
   const indicator = (
     <FlexBox className={cssClasses.INDICATOR_CONTAINER} justify={Justify.END}>
-      {status === "active" && hasDraft ? (
-        <DraftPencilIcon />
+      {!isRead ? (
```

Comment:
I'm pretty sure the last word I heard from Design was to prioritize the unread blue dot over the draft pencil icon, so I'm switching them around.